pkg/serverless/invocationlifecycle/trace.go

```diff
@@ -65,3 +84,27 @@ func endExecutionSpan(processTrace func(p *api.Payload), endTime time.Time) {
 		TracerPayload: tracerPayload,
 	})
 }
+
+func convertRawPayload(rawPayload string) (invocationPayload, bool) {
+	//Need to remove unwanted text and add } to unmarshal properly
+	var reg = regexp.MustCompile(`{(?:|(.*))*}`)
+	subString := reg.FindString(rawPayload)
+
+	payload := invocationPayload{}
+
+	err := json.Unmarshal([]byte(subString), &payload)
+	if err != nil {
+		log.Debug("Could not unmarshal the invocation event payload")
+		return payload, false
+	}
+
+	return payload, true
+}
+
+func convertStrToUnit64(s string) uint64 {
+	num, err := strconv.ParseUint(s, 0, 64)
+	if err != nil {
+		log.Debug("Error with string conversion of trace or parent ID")
+	}
+	return num
```

Comment:
what's inside num if the ParseUint fail? maybe a unit test against this behaviour would be good?